### DISCUSSION TITLE
Can you please create multiple looks per mob? (More info inside the subject)

### DISCUSSION THREAD

**Discussion by RogueAxolotl** [Draylar/identity#300] [Ideas]
As said in the title, can you please add more looks for certain mobs that usually have it in the game? As in, different axolotls will have the leucistic, wild, purple, gold, and cotton candy looks. Sheep would have different wool colors. I think this would be a fun addition, and I would most definitely use it. It would also do good for content creators (such as myself) in making little shorts and/or roleplays. I would 100% love this feature if it was added, so please consider it! Thank you for reading and have blessed day!!

**Comment by Draylar** [comment]
In progress locally, hoping to have it out soon. Just have to implement UI and finish up the API, so it might be a few weeks :)

**Reply by RogueAxolotl** [reply]
No problem. Thanks for the response!! Very excited for the update!! Also, thanks for a very amazing mod. This helped me out so much during one of my vidoes. **Keep up the great work**, Draylar! 👍

**Reply by Foxite** [reply]
@Draylar have you pushed your local progress anywhere? I'd like to get on this, but I don't want to waste time re-inventing your solution, especially since this seems like it requires a lot of changes to the code (as everything seems to assume only one identity per LivingEntity type).

**Reply by Draylar** [reply]
> @Draylar have you pushed your local progress anywhere? I'd like to get on this, but I don't want to waste time re-inventing your solution, especially since this seems like it requires a lot of changes to the code (as everything seems to assume only one identity per LivingEntity type).

I'll take another stab at it tonight and let you know how it goes. :)
edit: pushing working releases now with urgent bug fixes, I will still be looking at this later today

**Reply by Draylar** [reply]
![image](https://user-images.githubusercontent.com/17728338/170776133-bee32477-ec24-4dc2-9976-3e05631653f5.png)
![image](https://user-images.githubusercontent.com/17728338/170776163-c850d15a-f6b6-4b9f-a962-ec257a591a43.png)

👀

**Comment by Draylar** [comment]
Out in the next 2.3.0 update! :)

**Reply by Ch1216** [reply]
How do I become a different sheeps, I killed a yellow and white sheep but I only see a white sheep in the UI

**Reply by Draylar** [reply]
To confirm, you updated to the [2.3.0 binary released today](https://www.curseforge.com/minecraft/mc-mods/identity/files/3807262), correct? It should give you access to the Yellow Sheep entity in the UI after you kill one. Are you using the Fabric or Forge version?

**Reply by Natedogwoooof** [reply]
Hey I have a question why isn't there a different forms of like the Magma Cube Trader llama and villager and llama and mooshrom and horses

**Reply by Minecrafter142** [reply]
How do i become a type of villager, i kill a taiga librarian villager and get jobless plains villager in the UI

**Reply by Natedogwoooof** [reply]
Sadly you can't they should add the varents or make it randomized like zombie villager varents

**Comment by Natedogwoooof** [comment]
> To confirm, you updated to the [2.3.0 binary released today](https://www.curseforge.com/minecraft/mc-mods/identity/files/3807262), correct? It should give you access to the Yellow Sheep entity in the UI after you kill one. Are you using the Fabric or Forge version?
We are missing Lama and Trader Lama Varents
